CONTRIBUTORS.md

```diff
@@ -3,34 +3,36 @@ Chapel Contributors
 
 The following people have contributed to Chapel releases:
 
-Contributors to the Chapel 2.0 release
+Contributors to the Chapel 2.1 release
 --------------------------------------
 * Jade Abraham, [HPE]
-* Paul Cassella, [HPE]
+* [Dan Bonachea], [Lawrence Berkeley National Laboratory]
 * [Brad Chamberlain], [HPE]
 * Soohoon Choi, [HPE]
 * Sarah Coghlan, [HPE]
 * Jeremiah Corrado, [HPE]
+* Mark Dewing, individual contributor
 * Lydia Duncan, [HPE]
 * Daniel Fedorin, [HPE]
 * Michael Ferguson, [HPE]
 * Luca Ferranti, individual contributor
```

Comment:
@PHHargrove might now also qualify as a contributor with the merge of #25317

```suggestion
* Luca Ferranti, individual contributor
* [Paul H. Hargrove], [Lawrence Berkeley National Laboratory]
```

Reply:
Great point, thanks for pointing that out, Dan!  I'm going to do this on my branch rather than committing your suggestions since he's also listed in the older contributors section, so I need to remove him from there in a separate commit anyway (and don't think that was one of your suggested changes, though GitHub sometimes does hide things…).

Reply:
Of course, it _still_ took me two commits to get right…  🤦